test/cli/tarantool-leak.result

```diff
@@ -0,0 +1,5 @@
+TAP version 13
```

Comment:
Neither of the result files are necessary, because all of them contain a result in the TAP13 format: test-run knows how to check it.

Reply:
I add them because tests not passed without them.

Reply:
Evgeny re-checked and removed them.